unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/ContentProvider/ContentProvider.cs

```diff
@@ -109,9 +109,11 @@ public virtual string GetContentsUrl(string url)
             return null;
         }
 
-        public virtual bool TryGetContentsUrl_Raw(string url, out string result)
+        public virtual bool TryGetContentsUrl_Raw(string url, out string result, bool forceLowercaseURL = true)
         {
-            url = url.ToLower();
+            if (forceLowercaseURL)
+                url = url.ToLower();
```

Comment:
There are a few problems with this approach, let me elaborate:

Remember that this is needed to match with the mappings dictionary, so this check should be in `BakeHashes` if we aim to use the dictionary. The thing is, if we are targeting external URLs, mappings shouldn't we used, so maybe the correct approach is to skip the `ContentProvided` in that case.

Reply:
Yeap that's better, i didn't know we could avoid using the TryGetContentURL()